library/src/main/java/net/sparkminds/library/service/impl/BookManaServiceImpl.java

```diff
@@ -0,0 +1,230 @@
+package net.sparkminds.library.service.impl;
+
+import java.io.IOException;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import net.sparkminds.library.dto.bookmana.BookManaRequest;
+import net.sparkminds.library.dto.bookmana.BookManaResponse;
+import net.sparkminds.library.entity.Book;
+import net.sparkminds.library.enumration.EnumStatus;
+import net.sparkminds.library.exception.RequestException;
+import net.sparkminds.library.mapper.BookManaMapper;
+import net.sparkminds.library.repository.BookRepository;
+import net.sparkminds.library.service.BookManaService;
+import net.sparkminds.library.service.criteria.BookCriteria;
+import net.sparkminds.library.service.query.BookQueryService;
+import net.sparkminds.library.util.FileUtil;
+import tech.jhipster.service.filter.StringFilter;
+
+@Service
+@RequiredArgsConstructor
+@Log4j2
+public class BookManaServiceImpl implements BookManaService {
+	
+	private final MessageSource messageSource;
+	private final BookManaMapper bookManaMapper;
+	private final BookRepository bookRepository;
+	private final BookQueryService bookQueryService;
+	
+	@Value("${BASEURL_RESOURCES}")
+	private String BASEURL_RESOURCES;
```

Comment:
Chỗ đặt tên biến em follow theo camel case nha. Chỉ đặt in hoa khi nó là hằng số ( private static final)